<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EthoWatcherInpi</class>
 <widget class="QMainWindow" name="EthoWatcherInpi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>577</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>577</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>675</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>EthoWatcherOS: A Tool for Behavioral and Video-Tracking Analysis in Laboratory Animals</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourcefile.qrc">
    <normaloff>:/icons/icon_etw.JPG</normaloff>:/icons/icon_etw.JPG</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="imgCabecalho">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resourcefile.qrc">:/icons/ethoOS.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblNomeUsuario">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblUser">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbLoadUser">
         <property name="text">
          <string>Load User</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCreateUser">
         <property name="text">
          <string>Create User</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpConfiguracoes">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QStackedWidget" name="stackedPassos">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="pageIdentificacao">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QCheckBox" name="cbEthoDigi">
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Digital video-based ethography</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbEthoDigiTraking">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string> Tracking</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbEthoAna">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string> Tracking and digital video-based ethography</string>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageOpcoesEtografia">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Record/Analysis Options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QPushButton" name="pbBehavioralCatalog">
                <property name="text">
                 <string>Step 1: Create a behavioral catalog</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbRegisterVideo">
                <property name="text">
                 <string>Step 2: Register and format a video file</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbVideoEtho">
                <property name="text">
                 <string>Step 3: Start a video-based ethography and/or tracking</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbConfigAnalyses">
                <property name="text">
                 <string>Step 4: Data analysis</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageCalibracao">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QPushButton" name="pbTimeSegmentation">
                <property name="text">
                 <string>Segmentation of the data report by time periods</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbSegCompor">
                <property name="text">
                 <string>Segmentation of the data report by behavioral category</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbAnaliseSeque">
                <property name="text">
                 <string>Sequence analysis</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbAnliseConcor">
                <property name="text">
                 <string>Intra- &amp; Inter- observer reliability assessment</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="cbTutorEthoPrincipal">
             <property name="text">
              <string>Tutor EthoWatcjer Principal</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbTutorCadastroCatalogo">
             <property name="text">
              <string>Tutor Cadastro de Catalogo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbTutorCadastroUsuario">
             <property name="text">
              <string>Tutor Cadastro Usuario</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAvancar">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resourcefile.qrc">
           <normaloff>:/icons/old_go_next.ico</normaloff>:/icons/old_go_next.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonVoltar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resourcefile.qrc">
           <normaloff>:/icons/old_go_previous.ico</normaloff>:/icons/old_go_previous.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuSuperior">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem_Information">
    <property name="title">
     <string>System Information</string>
    </property>
    <addaction name="actionLicense_Agreement"/>
    <addaction name="actionAbout"/>
    <addaction name="actionControl_Tutor"/>
   </widget>
   <addaction name="menuSystem_Information"/>
  </widget>
  <action name="actionLicense_Agreement">
   <property name="text">
    <string>License Agreement</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionControl_Tutor">
   <property name="text">
    <string>Control Tutor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourcefile.qrc"/>
 </resources>
 <connections/>
</ui>
